CMakeLists.txt

```diff
@@ -309,34 +309,44 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
             RENAME ${filename})
     install(FILES ${icon_file}
             DESTINATION /usr/share/pixmaps
-            RENAME ${DESKTOP_ICON}.png)
+            RENAME ${icon_install_file})
   endfunction()
 
+  set(_icon_suffix)
+  if(CPACK_PACKAGE_SUFFIX)
+    if (${CPACK_PACKAGE_SUFFIX} STREQUAL "nightly" OR
+        ${CPACK_PACKAGE_SUFFIX} STREQUAL "unstable")
+      set(_icon_suffix ${CPACK_PACKAGE_SUFFIX})
+    endif()
+  endif()
+
   set( IMAGES_DIR ${CMAKE_CURRENT_SOURCE_DIR}/images )
   set(_app_name_suffix)
   if(CPACK_PACKAGE_SUFFIX)
     uppercase_first_letter(_app_name_suffix ${CPACK_PACKAGE_SUFFIX})
   endif()
   if(ENABLE_MANTIDPLOT)
     set(_icon_filename ${IMAGES_DIR}/mantidplot)
-    if(CPACK_PACKAGE_SUFFIX)
-      set(_icon_filename ${_icon_filename}${CPACK_PACKAGE_SUFFIX})
+    if(_icon_suffix)
+      set(_icon_filename ${_icon_filename}${_icon_suffix})
     endif()
     install_desktop_files(mantidplot${CPACK_PACKAGE_SUFFIX}.desktop
                           MantidPlot${_app_name_suffix}
                           ${CMAKE_INSTALL_PREFIX}/bin/MantidPlot
-                          ${_icon_filename}.png)
+                          ${_icon_filename}.png
+                          mantidplot${CPACK_PACKAGE_SUFFIX}.png )
```

Comment:
So the package builds now but we don't have icon files `mantidplot41.png`. Is that a problem?

Reply:
This change copies `mantidplot.png` to be `mantidplot41.png`